README.md

```diff
@@ -250,29 +275,29 @@ spec:
       unmanaged: true
 ```
 
-## Deploying in cluster
+### Deploying in cluster
 
- 1. export IMG=quay.io/dgoodwin/cloud-credential-operator:latest
-   * You can upload to a personal repo if you wish to build images from source.
- 1. make buildah-push
- 1. make deploy
+ 1. `export IMG=quay.io/dgoodwin/cloud-credential-operator:latest`
+    * You can upload to a personal repo if you wish to build images from source.
+ 1. `make buildah-push`
+ 1. `make deploy`
 
-Cred Minter should now be running in openshift-cloud-credential-operator.
+Cred Minter should now be running in the `openshift-cloud-credential-operator` namespace.
 
-# Credentials Requests
+## Credentials Requests
 
-The primary custom resource used by this operator is the CredentialsRequest, which allows cluster components to request fine grained credentials.
+The primary custom resource used by this operator is the CredentialsRequest, which allows cluster components to request fine-grained credentials.
 
-CredentialRequests spec consist of:
- 1. secretRef - Points to the secret where the credentials should be stored once generated. Can be in a separate namespace from the CredentialsRequest where it can be used by pods. If that namespace does not yet exist, the controller will immediately sync when it is created.
- 2. providerSpec - Contains the [cloud provider specific credentials specification](https://github.com/openshift/cloud-credential-operator/tree/master/pkg/apis/cloudcredential/v1).
+A CredentialRequest spec consists of:
+ 1. `secretRef` - Points to the secret where the credentials should be stored once generated. Can be in a separate namespace from the CredentialsRequest where it can be used by pods. If that namespace does not yet exist, the controller will immediately sync when it is created.
+ 2. `providerSpec` - Contains the [cloud provider specific credentials specification](pkg/apis/cloudcredential/v1).
 
-Once created, assuming admin credentials are available, the controller will provision a user, access key, and user policy in AWS. The access and secret key will be stored in the target secret specified above.
+Once created, assuming admin credentials are available, the controller will provision e.g. a user, access key, and user policy in AWS. The access and secret key will be stored in the target secret specified above.
```

Comment:
`e.g. ` do we need it?  the following statement looks correct to me
`Once created, assuming admin credentials are available, the controller will provision a user, access key, and user policy in AWS.`

Reply:
The way I read it, what happens in AWS is one example of how this works. Against other cloud providers, the specific artifacts provisioned may be different. No?

Reply:
ahh yes